lib/AST/DeclContext.cpp

```diff
@@ -1285,7 +1285,7 @@ SourceLoc swift::extractNearestSourceLoc(const DeclContext *dc) {
   case DeclContextKind::SerializedLocal:
     return extractNearestSourceLoc(dc->getParent());
   }
-  llvm_unreachable("Unhandled DeclCopntextKindin switch");
+  llvm_unreachable("Unhandled DeclContextKindin switch");
```

Comment:
not sure if the `in` needs to be part of the token or could be its own word.

Reply:
I think this one should be its own word